pages/1.12/cli/command-reference/index.md

```diff
@@ -8,20 +8,34 @@ excerpt: Exploring the commands unique to DC/OS
 enterprise: false
 ---
 
-
 This section lists the unique DC/OS commands.
 
+# Base commands
+
+| Command | Description |
+|---------|-------------|
+| [dcos](/1.12/cli/command-reference/dcos/)   | The command line for DC/OS. |
+| [dcos auth](/1.12/cli/command-reference/dcos-auth/)   |  Manage DC/OS identity and access. |
+| [docs cluster](/1.12/cli/command-reference/dcos-cluster/) |  Manage connections to DC/OS clusters. |
+| [dcos config](/1.12/cli/command-reference/dcos-config/) |  Manage the DC/OS configuration file. |
+| [dcos help](/1.12/cli/command-reference/dcos-help/)    | Display help information about DC/OS.  |
+| [dcos plugin](/1.12/cli/command-reference/dcos-plugin/) | Install and manage DC/OS CLI plugins. |
+
+# Core plugin commands
+
+| Command | Description |
+|---------|-------------|
+| [dcos job](/1.12/cli/command-reference/dcos-job/)    | Deploy and manage jobs in DC/OS.  |
+| [dcos marathon](/1.12/cli/command-reference/dcos-marathon/)  |  Deploy and manage applications to DC/OS.  |
+| [dcos node](/1.12/cli/command-reference/dcos-node/)   |  Administer and manage DC/OS cluster nodes.  |
+| [dcos package](/1.12/cli/command-reference/dcos-package/) | Install and manage DC/OS software packages. |
+| [dcos service](/1.12/cli/command-reference/dcos-service/)  |  Manage DC/OS services.  |
+| [dcos task](/1.12/cli/command-reference/dcos-task/)  |  Manage DC/OS tasks.  |
+
+# Enterprise plugin commands
```

Comment:
If this section is specific to "Enterprise only" then use Enterprise tag. 
Replace # Enterprise plugin commands  with # Enterprise plugin commands [enterprise type="inline" size="small" /].